Resources/doc/getting_started/installation.rst

```diff
@@ -98,6 +98,17 @@ line in the `app/AppKernel.php` file of your project:
 
         $ bower install ./vendor/sonata-project/admin-bundle/bower.json
 
+.. note::
+
+	You must enable translator service. Uncomment the translator configuration node in app/config/config.yml
+	
+		.. code-block:: bash
+		
+			framework:
```

Comment:
Not sure to have done what you mean. I just remove space between key and value.
May be it is my editor that give wrong space. Can you be more specific?